packages/mdc-menu/index.ts

```diff
@@ -151,50 +140,41 @@ class MDCMenu extends MDCComponent {
     }
   }
 
-  /** @param {boolean} quickOpen */
-  set quickOpen(quickOpen) {
-    this.menuSurface_.quickOpen = quickOpen;
-  }
-
-  /** @param {boolean} isFixed */
-  setFixedPosition(isFixed) {
+  setFixedPosition(isFixed: boolean) {
     this.menuSurface_.setFixedPosition(isFixed);
   }
 
   hoistMenuToBody() {
     this.menuSurface_.hoistMenuToBody();
   }
 
-  /** @param {boolean} isHoisted */
-  setIsHoisted(isHoisted) {
+  setIsHoisted(isHoisted: boolean) {
     this.menuSurface_.setIsHoisted(isHoisted);
   }
 
-  /**
-   * @param {number} x
-   * @param {number} y
-   */
-  setAbsolutePosition(x, y) {
+  setAbsolutePosition(x: number, y: number) {
     this.menuSurface_.setAbsolutePosition(x, y);
   }
 
   /**
    * Sets the element that the menu-surface is anchored to.
-   * @param {!HTMLElement} element
    */
-  setAnchorElement(element) {
+  setAnchorElement(element: Element) {
     this.menuSurface_.anchorElement = element;
   }
 
   handleAfterOpened_() {
     const list = this.items;
     if (list.length > 0) {
-      list[0].focus();
+      (list[0] as HTMLElement).focus();
     }
   }
 
-  /** @return {!MDCMenuFoundation} */
+  /**
+   * @return {!MDCMenuFoundation}
```

Comment:
change to `@return MDCMenuFoundation`

Reply:
Removed comment in favor of TypeScript syntax